test/pjrt/test_experimental_pjrt_multi_cpu.py

```diff
@@ -108,6 +108,23 @@ def test_spawn(self, spawn):
     indices = sorted(queue.get(block=False) for _ in range(queue.qsize()))
     self.assertListEqual(indices, list(range(4)))
 
+  @staticmethod
+  def _hlo_dump(tmpdir: str):
+    os.environ['XLA_SAVE_TENSORS_FMT'] = 'hlo'
+    os.environ['XLA_SAVE_TENSORS_FILE'] = os.path.join(tmpdir, 'save.hlo')
+
+    x = torch.randn((3, 3), device=xm.xla_device())
+    xm.mark_step()
+    x.cpu()
+
+  def test_hlo_dump(self):
+    tmpdir = self.create_tempdir().full_path
+    pjrt._run_multiprocess(self._hlo_dump, tmpdir)
+
+    files = os.listdir(tmpdir)
+    for i in range(4):
+      self.assertIn(f'save.hlo.{i}', files)
```

Comment:
do you need to clean up the hlo dump at the end of the test?

Reply:
No. We got the temp dir from the test class, which cleans up temp files at the end